test2code/src/main/kotlin/com/epam/drill/test2code/coverage/CoverageSender.kt

```diff
@@ -115,3 +107,22 @@ class IntervalCoverageSender(
         }
     }
 }
+
+private fun processTotalSizeByte(logger: KLogger): BigInteger {
+    try {
+        val retentionLimit = JvmModuleConfiguration.getCoverageRetentionLimit()
+        val maxMemory = Runtime.getRuntime().maxMemory()
+        if (maxMemory == Long.MAX_VALUE) {
+            return DataSize.parse(retentionLimit)
+                .toUnit(ByteUnit.BYTE)
+                .value
+                .toBigInteger()
+        }
+        logger.warn { "CoverageRetentionLimit may be skipped." }
```

Comment:
"maybe" is a poor help for a user + it will _always_ print here. What is the point?